### PR TITLE
Simplified Chinese Support

### PR DESCRIPTION
**Pull request by fairycn**
Simplified Chinese Support #187

### CODE REVIEW

**Comment by MrGreensWorkshop** [comment]
@fairycn Thank you so much for your time and effort. I wasn't expecting to get new language support in such a short time. I reviewed your PR; there are some issues that need to be addressed, and I will add comments to that effect. Let's work together and fix those issues. Thank you for your patience.

File: README-zh-cmn-Hans.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+[简体中文]  [[English](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/README.md#readme)] [[日本語](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/README-jp.md#readme)] [[Português do Brasil](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/README-pt.md#readme)]
```
Comment (MrGreensWorkshop) [inline README-zh-cmn-Hans.md:1]: - Readme filename needs to be renamed to README-zh.md (based on ISO_639-1 https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)
- `[简体中文]` needs to be after `Português`.

File: README-zh-cmn-Hans.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+[<img src="https://img.shields.io/github/stars/MrGreensWorkshop/MrGreen-JekyllTheme" alt="GitHub stars" data-no-image-viewer>](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/README.md#readme)
+[<img src="https://img.shields.io/github/license/MrGreensWorkshop/MrGreen-JekyllTheme" alt="GitHub license" data-no-image-viewer>](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/blob/main/LICENSE.txt)
+[<img src="https://shields.io/badge/Github%20Sponsors-Support%20me-blue?logo=GitHub+Sponsors" alt="Support me on GitHub Sponsors" data-no-image-viewer>](https://github.com/sponsors/MrGreensWorkshop "Support me on GitHub Sponsors")
+[<img src="https://shields.io/badge/Patreon-Support%20me-blue?logo=Patreon" alt="Support me on Patreon" data-no-image-viewer>](https://patreon.com/MrGreensWorkshop "Support me on Patreon")
+[<img src="https://shields.io/badge/Ko--fi-Tip%20me-blue?logo=kofi" alt="Tip me via Ko-fi" data-no-image-viewer>](https://ko-fi.com/MrGreensWorkshop "Tip me via Ko-fi")
+
```
Comment (MrGreensWorkshop) [inline README-zh-cmn-Hans.md:14]: These are needs to be not translated.
- Support me on GitHub Sponsors
- Support me on Patreon
- Tip me via Ko-fi

**Comment by fairycn** [comment]
What problem has not been found in the current test? the downside is that using standard language abbreviations is too long. I think it is better to describe it directly in the relevant language, such as "简体中文".

**Comment by MrGreensWorkshop** [comment]
@fairycn Hi, I moved everything into this  [branch](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/tree/feature_chinese_language_support). There are some non-translated texts that need some work. Whenever you want to continue, you are always welcome. Please submit your PRs to this [branch](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/tree/feature_chinese_language_support).  (btw, I deleted some confusing messages from issues.)

Thanks in advance.

**Pull request merged**